dist/privacy-sandbox.js

```diff
@@ -0,0 +1,402 @@
+
+
+    // Checking if the request header includes 'Attribution-Reporting-Eligible' to initiate the registration of source or trigger
+    if (reqHeaders.has('attribution-reporting-eligible')) {
+      result['attributionReportingAPI']['attributionReportingEligibleHeader']['sentTo'].push(requestDomain);
+      await apiCallerAdd(requestDomain);
+    }
+
+    // Checking if the response header includes
+    // 'Attribution-Reporting-Register-Source' or
+    // 'Attribution-Reporting-Register-Trigger' to complete registration of
+    // source or trigger Source registration happens on seller (e.g., publisher)
+    // website where impression is registered and trigger registration happens
+    // on buyer (e.g., advertiser) website where conversion completes. Each
+    // entry in
+    // result['attributionReportingAPI']['completedRegistrations']['AttributionReportingRegisterSourceHeader']
+    // is represented as {requestDomain: {"destination": "", "eventEpsilon": 0}}
+    // Higher the epsilon, the more the privacy protection
+    if (respHeaders.has('attribution-reporting-register-source')) {
+      jsonString = respHeaders.get('attribution-reporting-register-source');
+      const { destination, event_level_epsilon } = JSON.parse(jsonString);
+      if (!result['attributionReportingAPI']['completedRegistrations']['AttributionReportingRegisterSourceHeader'][requestDomain]) {
+        result['attributionReportingAPI']['completedRegistrations']['AttributionReportingRegisterSourceHeader'][requestDomain] = [];
+      }
+      result['attributionReportingAPI']['completedRegistrations']['AttributionReportingRegisterSourceHeader'][requestDomain].push({ "destination": destination, "eventEpsilon": event_level_epsilon });
+      await apiCallerAdd(requestDomain);
+    } else if (respHeaders.has('attribution-reporting-register-trigger')) {
+      result['attributionReportingAPI']['completedRegistrations']['AttributionReportingRegisterTriggerHeader'].push(requestDomain);
+      await apiCallerAdd(requestDomain);
+    }
+
+    /***************************************************************************
+     * Federated Credentials Manager
+     * Documentation:
+     * Test site(s):
+     **************************************************************************/
+
+    //Todo
+
+
+    /***************************************************************************
+     * Fenced Frames
+     * Documentation:
+     * Test site(s):
+     **************************************************************************/
+
+    //Todo
+
+    /***************************************************************************
+     * FLoC (deprecated - are some still calling it?))
+     * Documentation: https://web.dev/articles/floc
+     * Test site: https://floc.glitch.me/
+     **************************************************************************/
+    if (checkResponseBody(request, 'document.interestCohort')) {
```

Comment:
@yohhaan [Implemented currently](https://github.com/HTTPArchive/custom-metrics/blob/fc9f4f4f7cf7220164c556216b2cf067631887ed/dist/privacy.js#L178). But planned for deprecation in [this PR](https://github.com/HTTPArchive/custom-metrics/pull/117).
Do you want to leave it moving forward, or historical data is enough?

Reply:
Even though the API is deprecated it could be nice to leave the detection on to see if code not up to date is still calling it.

Reply:
@yohhaan Do you want to keep page level metric as it was implemented in privacy.js for 2024 analysis?

Reply:
You can deprecate the page level metric, and we can just keep the detection in privacy-sandbox.js